x-pack/packages/security/authorization_core/src/actions/ui.ts

```diff
@@ -40,4 +40,12 @@ export class UIActions implements UIActionsType {
 
     return `${this.prefix}${featureId}/${uiCapabilityParts.join('/')}`;
   }
+
+  /**
+   * Checks if the action is a valid UI action.
+   * @param action The action string to check.
+   */
+  public isValid(action: string) {
```

Comment:
__note:__ it's necessary to filter out non-UI actions when complementing actions of the registered deprecated features with the UI actions of the replacement features. This ensures that user roles with deprecated privileges can toggle capabilities exposed by the replacement privileges.